### DISCUSSION TITLE
UART on trigorilla 3.0.6

### DISCUSSION THREAD

**Discussion by dDaYb** [1coderookie/KobraGoNeoInsights#9] [General]
Hi,
Great guide! Thanks !

Do you know is it possible to use swdio and swclk as uart? 
Is there a datashit for trigorilla 3.0.6?

I am looking how to connect raspberry to kobra neo without usb cable))

**Comment by 1coderookie** [comment]
Hi,
happy to hear that - if you like it, please consider starring the repo then ;)  
I'm trying to figure out the same since a longer time (not for a connection of a RPi but for sth different). I don't know tho, there isn't any datasheet or circuit diagram available. I've reached out to AC as well - nothing.. :(  
Imho it should be possible, but this is just a guess, so everything you do is on your own risk.  
IF it happens that you do something on that connector, please report back!
